src/core/etl/src/Flow/ETL/Loader/BatchLoader.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Loader;
+
+/**
+ * Loaders implementing BatchLoader interface points that they can use "batch size" optimization.
+ */
+interface BatchLoader
+{
+}
```

Comment:
I'm not a huge fan of empty interfaces without any behavior, it just feels wrong

Reply:
Yeah, true to that, but other ways:
- extend the normal `Loader` interface, but this will still be empty...
- add some method returning a boolean, but sounds smelly too,
- add a new method in the `Loader` interface returning boolean, ie. `supportsBatchLoading() : bool`

Reply:
Another problem is that we also have batching pipelines that are being excluded here and that are also stored as a list of classes. For them, we would need to create another empty interface. 
I think I'm better keeping those classes as they are now